### ISSUE TITLE
attempt-backport should ignore semver-major changes

### ISSUE THREAD

**Issue by mscdex** [nodejs/github-bot#105]
Title says it all, github-bot should not attempt to backport changes marked as semver-major.

**Comment by Fishrock123** [comment]
Should be fairly easy to do, but a lot will slip by because they will be labeled by us later so I'm not 100% sure what to do about that. Have any ideas?

**Comment by targos** [comment]
> a lot will slip by because they will be labeled by us later

Yes but at least once the PR is marked semver-major, the bot will stop trying on every push.

**Comment by phillipj** [comment]
Maybe remove any previous backporting labels when GitHub emits a labelling event for semver-major being added?

**Comment by Fishrock123** [comment]
@TheAlphaNerd Think it might work better if we only added these labels once a PR is closed/merged?

**Comment by Fishrock123** [comment]
That does mean that people won't know if they'll need to adjust it until after though... :/ So IMO would be worse.

**Comment by TheAlphaNerd** [comment]
I'm ok with this. As long as it is accurate.

**Comment by mscdex** [comment]
So does everyone agree that the bot should (at least) not add any backport-related labels if it sees a change in commits and a `semver-major` is set on the PR?

I'm not sure proactively removing existing backport-related labels would be a good thing, especially if someone intentionally added the label.

**Comment by Fishrock123** [comment]
@mscdex that is why https://github.com/nodejs/github-bot/pull/104 only removes it if the bot (last?) added it.
